plugins/nf-azure/src/test/nextflow/cloud/azure/file/AzBashLibTest.groovy

```diff
@@ -44,20 +50,20 @@ class AzBashLibTest extends Specification {
                 ret=$(azcopy cp "$source?$AZ_SAS" "$target" 2>&1) || {
                     ## if fails check if it was trying to download a directory
                     mkdir -p $target
-                    azcopy cp "$source/*?$AZ_SAS" "$target" --recursive >/dev/null || {
+                    azcopy cp "$source/*?$AZ_SAS" "$target" --recursive --check-md5 $_AZCOPY_CHECK_MD5 --overwrite $_AZCOPY_OVERWRITE >/dev/null || {
                         rm -rf $target
                         >&2 echo "Unable to download path: $source"
                         exit 1
                     }
                 }
             }
-            '''.stripIndent()
+            '''.stripIndent().replaceAll(' ', '')
```

Comment:
Why there's the need for `.replaceAll(' ', '')`? never needed this before

Reply:
Ah, this is the problem related to the large string comparisons while testing, Jorge lent his experience here and this tweak makes Spock happy with the string comparison.

Reply:
hahaha, well in fact was a trick to use during dev in order to know where the difference comes but once solved and test is green can be removed

Reply:
Ah, I thought this was meant for the long run!

With the use of `replaceAll` in first and second tests, both pass. Okay, the first test doesn't pass without `replaceAll`, but for the second I've removed this.